docs/source/cuvs_bench/build.rst

```diff
@@ -0,0 +1,59 @@
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Build cuVS Bench From Source
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Dependencies
+============
+
+CUDA 11 and a GPU with Pascal architecture or later are required to run the benchmarks.
```

Comment:
Do we still support pascal gpus ? https://docs.rapids.ai/notices/rsn0034/

Reply:
Nope, fixed.